sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -367,6 +376,21 @@ def fit(self, X, y, sample_weight=None):
 
             .. versionadded:: 0.23
 
+        X_val : array-like of shape (n_val, n_features)
+            Additional sample of features for validation used in early stopping.
```

Comment:
the doc here deserves a note on lack of transformations on these parameters if used in a pipeline.

Also probably a mini section in the user guide.